### PR TITLE
Updating CommonVoice CTC English and Conformer English

### PR DESCRIPTION
**Pull request by TParcollet**
The CTC en recipes and Conformer EN recipes were way out of date. 
Now we have something a bit better.
Models are training, i'll update the results.

- [x] update code
- [x] train models
- [x] verify dynamic batching does not break legacy code
- [x] update readme with results

### CODE REVIEW

**Comment by TParcollet** [comment]
@Adel-Moumen is ready to review and merge. The conformer WER is not there yet, but I'll edit the readme directly on the main repo once the decoding is done :p

File: recipes/CommonVoice/ASR/transformer/train.py [hunk 1]
```diff
@@ -269,15 +269,21 @@ def dataio_prepare(hparams, tokenizer):
...
         csv_path=hparams["test_csv"],
         replacements={"data_root": data_folder},
     )
 
     # We also sort the validation data so it is faster to validate
-    test_data = test_data.filtered_sorted(sort_key="duration")
+    test_data = test_data.filtered_sorted(
+        sort_key="duration",
+        key_max_value={"duration": hparams["avoid_if_longer_than_val_test"]},
+    )
```
Comment (Adel-Moumen) [inline recipes/CommonVoice/ASR/transformer/train.py:286]: I think we should definitely add a warning/log saying that we are skipping files that have a duration > to `avoid_if_longer_than_val_test`.

**Review by Adel-Moumen — COMMENTED** [review]
hi, overall lgtm. I leaved one comment about `avoid_if_longer_than_val_test`.

File: recipes/CommonVoice/ASR/transformer/train.py (hunk 1 above)
Comment (TParcollet) [reply recipes/CommonVoice/ASR/transformer/train.py:286]: Do it :p

**Comment by TParcollet** [comment]
I merge cuz I'm the boss blblblblb.

**Pull request merged**